radar_control/radar_control.py

```diff
@@ -399,6 +385,14 @@ def radar():
                     time_remains = True
                     integration_period_done_time = integration_period_start_time + \
                         timedelta(milliseconds=(float(aveperiod.intt)))  # ms
+
+                    if __debug__:
+                        for sequence_index, sequence in enumerate(aveperiod.sequences):
+                            write_samples_to_file(experiment.txrate,
+                                                  experiment.txctrfreq,
+                                                  sequence_dict_list[sequence_index],
```

Comment:
missing all-repeats boolean here

Reply:
it writes the entire sequence now so this is not needed anymore.

Reply:
This has changed as well now I think.